examples/nextjs-with-typescript/src/theme.ts

```diff
@@ -14,6 +22,9 @@ const theme = createTheme({
       main: red.A400,
     },
   },
+  typography: {
+    fontFamily: roboto.style.fontFamily,
+  },
```

Comment:
```suggestion
```

We can probably drop this, as it is the default value: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createTypography.js#L10

Reply:
The generated Roboto font will have some hash part at the end so the applied Roboto fontFamily name will actually be something like `__Roboto_123456 ` which is why we need to set it to the theme